ddtrace/monkey.py

```diff
@@ -79,19 +93,18 @@ def patch(raise_errors=True, **patch_modules):
     """
     modules = [m for (m, should_patch) in patch_modules.items() if should_patch]
     for module in modules:
-        # TODO: this is a temporary hack until we shift to using
-        # post-import hooks for everything.
-        if module == 'celery':
-            # if patch celery via post-import hooks
-            from wrapt.importer import when_imported
-
-            @when_imported('celery')
-            def patch_celery(hook):
-                from ddtrace.contrib.celery import patch
-                patch()
-
-            # manually add celery to patched modules
-            _PATCHED_MODULES.add(module)
```

Comment:
this line was important to keep as it enables debug information to be consistent with the previous patching approach. it's ugly to keep, but we need to stay consistent until this whole patching system is reworked (soon, hopefully!!) 

by removing it our logging for a celery application would look like:

```
INFO:ddtrace.monkey:patched 2/23 modules (flask,sqlite3)
```

(we're missing celery)

Reply:
Added this back in